src/main/java/codeu/controller/ProfileServlet.java

```diff
@@ -166,10 +167,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     if(userId.equals(profileId)) {
       // User is viewing their own profile page
       String about = request.getParameter("editAbout");
-      userStore.getUser(userId).setAbout(about);
+      user.setAbout(about);
+      userStore.putUser(user);
       request.getSession().setAttribute("about", about);
     }
 
-    response.sendRedirect("/profile/" + userId.toString());
-  }
+    String url = "/profile/" + userId.toString();
+    response.sendRedirect(url);
```

Comment:
Why separate the original line into two?

Reply:
It was just something leftover from debugging!